packages/Toast/ToastWrapper.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { cloneElement } from 'react'
+import toastRHT, { Toast, ToastPosition } from 'react-hot-toast/headless'
+
+import * as S from './styles'
+
+type ToastWrapperProps = {
+  calculateOffset: (
+    toast: Toast,
+    opts?:
```

Comment:
```
opts?: {
  reverseOrder?: boolean
  gutter?: number
  defaultPosition?: ToastPosition
}
```

should be already `undefined` with the `?:`

Reply:
Yes you're right 😄